tools/unitt-tester.art

```diff
@@ -0,0 +1,3 @@
+import {unitt}!
```

Comment:
```suggestion
import.version:1.1.2 {unitt}!
```

By doing it like this, if the package is not installed, it will be - we don't have to install it ourselves manually! (= one of the advantages of doing it the... Ruby "gem" way 😉 )